### PR TITLE
ncm-filesystems: documentation cleanup

### PR DESCRIPTION
**Pull request by wdpypere**
Mainly removed the TODO list. I will create github issues for them.

### CODE REVIEW

File: ncm-filesystems/src/main/perl/filesystems.pod [hunk 1]
```diff
@@ -12,33 +11,23 @@ to create and remove blockdevices without restarting or
...
+and creates new filesystems, if needed.
 
 A blockdevice is useful only for its ability to hold a
 filesystem. Blockdevices with no filesystems associated will not be
 created. If you want any such device, create a filesystem with
-"type"="none" and "mount"=false.
-
-=head2 To do list
-
-It will also remove filesystems and blockdevices that are not listed
```
Comment (ned21) [inline ncm-filesystems/src/main/perl/filesystems.pod]: I don't think this is just a TODO, it's also an important piece of functionality. The second sentence "For this to be safe, ..." can be removed. Should it be replaced by a link to the GH issue number?

Comment (wdpypere) [inline ncm-filesystems/src/main/perl/filesystems.pod]: Good point, I'll reinsert it as a warning note. I can insert the GH issue as well if you want but I don't think documentation is the correct place for that.

**Comment by wdpypere** [comment]
@ned21 remark addressed.

**Comment by jrha** [comment]
Could you reference this PR in the issues you created?

**Comment by wdpypere** [comment]
@jrha will do

**Comment by jrha** [comment]
Thanks!

**Pull request merged**